service/controller.go

```diff
@@ -14,6 +19,65 @@ const (
 	hardcodeShare = "fs-ht"
 )
 
+type Volume_Type int32
+
+const (
+	Volume_Type_Unknow Volume_Type = 0
+	Volume_Type_Block  Volume_Type = 1
+	Volume_Type_File   Volume_Type = 2
+)
+
+var (
+	// BlockVolumePrefix is a prefix of Block volume's ID.
+	BlockVolumePrefix = fmt.Sprintf("%s-block-", Name)
+	// FileVolumePrefix is a prefix of File volume's ID.
+	FileVolumePrefix = fmt.Sprintf("%s-file-", Name)
+)
+
+// getBackendIdAndTypeByVolumeId is used to get id and volume type by volume id
+// if it is block type, backend id is block id in backend
```

Comment:
in-consistent with the implementation?

Reply:
Where is in-consistent? This method is used to remove the prefix of volume id .
If it's block type, this method returns block_id (lun_id).
If it's file type, this method returns file_id:share_id.